static/app/components/group/suggestedOwners/ownershipRules.tsx

```diff
@@ -175,7 +175,7 @@ const Header = styled('h6')`
 `;
 
 const Content = styled('span')`
-  color: ${p => p.theme.gray500};
+  color: ${p => p.theme.textColor};
```

Comment:
Should it be `textColor` or `subText`

Reply:
@evanpurkhiser `textColor` matches the `gray500` for the lightAlias. The design used `gray500` for the content.